.github/workflows/multi-gpu-e2e.yml

```
@@ -30,26 +30,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include:
-          #          - cuda: 129
-          #            cuda_version: 12.9.1
-          #            python_version: "3.12"
-          #            pytorch: 2.9.1
-          #            axolotl_extras: "fbgemm-gpu"
-          #            num_gpus: 2
-          #            dockerfile: "Dockerfile-uv.jinja"
           - cuda: 130
```

actionlint (1.7.12)

[error] "matrix" section is sequence node but mapping node is expected

(syntax-check)